src/video_core/amdgpu/liverpool.cpp

```diff
@@ -20,6 +20,20 @@ static const char* acb_task_name{"ACB_TASK"};
 
 std::array<u8, 48_KB> Liverpool::ConstantEngine::constants_heap;
 
+static std::span<const u32> checked_subspan(std::span<const u32> span, size_t offset) {
```

Comment:
```suggestion
static std::span<const u32> NextPacket(std::span<const u32> span, size_t offset) {
```
Shorter name and I feel more expressive of what function does